### PR TITLE
Improved grammar and sentence structure

### PR DESCRIPTION
**Pull request by jacoblukewood**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,8 @@
 # PyFun-Backend
-This is a backend for Python education for YunOSC writeing in Python.  
-Frontend is write in Vue2, please reference at [here](https://github.com/YunOSC/PyFun-Frontend).
+This is a backend for Python education for YunOSC writing in Python.  
+Frontend is written in Vue2. A reference to Vue2 can be found [here](https://github.com/YunOSC/PyFun-Frontend).
 
 # Contributing
-If you want to add more stage or lessons.
-Reference at [here](https://github.com/YunOSC/PyFun-Backend/blob/master/CONTRIBUTING.md).
+If you would like to add to add more stage or lessons a you can find more information about contributing [here](https://github.com/YunOSC/PyFun-Backend/blob/master/CONTRIBUTING.md).
```
Comment (jackey8616) [inline README.md]: Oops. `to add to add`

Comment (jacoblukewood) [reply README.md]: Good catch!

File: README.md [hunk 2]
```diff
@@ -3,6 +3,6 @@ This is a backend for Python education for YunOSC writing in Python.
 Frontend is written in Vue2. A reference to Vue2 can be found [here](https://github.com/YunOSC/PyFun-Frontend).
 
 # Contributing
-If you would like to add to add more stage or lessons a you can find more information about contributing [here](https://github.com/YunOSC/PyFun-Backend/blob/master/CONTRIBUTING.md).
+If you would like to add more stage or lessons a you can find more information about contributing [here](https://github.com/YunOSC/PyFun-Backend/blob/master/CONTRIBUTING.md).
```
Comment (jackey8616) [inline README.md]: `lessons a you can find more` <--- I don't know what did I type..
Can you help me make it more smoothly ?

**Comment by jackey8616** [comment]
Invalid label will remove after the PR been accepted.

**Pull request closed without merging**